### PR TITLE
Fix a bug in get_pos_status

### PR DESCRIPTION
**Pull request by MarkYu98**
Error when agent_index is 0

### CODE REVIEW

File: pygame_rl/scenario/soccer_environment.py [hunk 1]
```diff
@@ -467,7 +467,7 @@ def __init__(self, env_options=None, renderer_options=None):
 
     def reset(self):
         self.state.reset()
-        self._set_weights()
+https://www.cnblogs.com/artesian0526/p/9267906.html        self._set_weights()
```
Comment (sc420) [inline pygame_rl/scenario/soccer_environment.py]: Please remove the link

File: pygame_rl/scenario/soccer_environment.py [hunk 2]
```diff
@@ -489,7 +489,7 @@ def _set_weights(self):
                     self.state.set_agent_mode(agent_index, 'OFFENSIVE')
                 # Set the weights for each mode
                 self.weights['ADVANCE'] = 1.0 - self.defensiveness
-                self.weights['INTERCEPT'] = 1.0 - self.defensiveness
+                self.weights['INTERCEPT'] = 1.0 - self.defensivenesshttps://www.cnblogs.com/artesian0526/p/9267906.html
```
Comment (sc420) [inline pygame_rl/scenario/soccer_environment.py]: Please remove the link

**Comment by sc420** [comment]
Thank you for fixing the bug! But there seem to have some links accidentally added in the code, please remove them and commit again. Thanks!

**Comment by MarkYu98** [comment]
Sorry, don't sure where the links come from. Removed.

**Comment by sc420** [comment]
No problem, it's merged. Thanks!

**Pull request merged**